tasks/setup/InfracostSetup/index.ts

```diff
@@ -0,0 +1,284 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+import * as toolLib from 'azure-pipelines-tool-lib/tool';
+import * as path from 'path';
+import * as os from 'os';
+import * as semver from 'semver';
+import fetch from 'node-fetch';
+
+/**
+ * archMappings defines how different https://nodejs.org/api/os.html#os_os_arch
+ * should be represented in terms of the infracost cli assets.
+ */
+const archMappings: { [s: string]: string } = {
+  x64: 'amd64',
+};
+
+/**
+ * mapArch returns a string of the correct binary arch to use in the pipeline.
```

Comment:
Just FYI, caught my eye in [Airbnb styleguide](https://github.com/airbnb/javascript#comments--spaces), not sure if we want to do the same here. Leaving for reference.

```suggestion
 * mapArch() returns a string of the correct binary arch to use in the pipeline.
```

Reply:
sounds good